monkestation/code/modules/art_sci_overrides/artifact_components/_artifact_effect.dm

```diff
@@ -0,0 +1,66 @@
+/datum/artifact_effect
```

Comment:
for clarity sake for now and the future move the arifact_effect and its subtypes into a folder of its own so we can easily find them later

Reply:
Allright, done and done.